### PR TITLE
Updated README.md please don't open review again

### PR DESCRIPTION
**Pull request by UddhavNavneeth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+We have incorporated the model given to us with Core ML of swift to create a feature which instantly detects the device in front and adds it to the collection of devices.  
+
+2. Detecting Devices on a Particular Network  
+We have created a feature similar to Fing which lists out the IP address and MAC address of all the devices on a particular network.  
+
+3. Creating an algorithm to uniquely identify each device  
+We have combined a devices MAC address alongwith a randomly generated value in backend to create a unique ID.
+
+# Team
+Sarvad - IOS
```
Comment (s4rv4d) [inline README.md]: iOS*
and instead of writing names directly add our GitHub handles

Comment (s4rv4d) [reply README.md]: add your , mine and diptanshus GitHub handle

Comment (s4rv4d) [reply README.md]: and its not IOS you disrespectful piece of shit, its iOS

**Review by s4rv4d — CHANGES_REQUESTED** [review]

**Comment by UddhavNavneeth** [comment]
Please check now idiot

**Pull request merged**